content/about/faq/_index.md

```diff
@@ -1,6 +1,6 @@
 +++
 title = "Frequently Asked Questions"
-date = "2019-03-28"
+date = "Last Updated 6/9/20"
```

Comment:
Why would you switch from an unambiguous date format to an ambiguous one, and then pick the American version?